### PR TITLE
add a few convex models

### PR DESCRIPTION
**Pull request by dpo**
models implemented by @DahitoMA

### CODE REVIEW

File: src/brownden.jl [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#   ACM Transactions on Mathematical Software, vol. 7(1), pp. 17-41, 1981.
+#
+#   See also Buckley#30
+#
+#   classification SUR2-AN-4-0
+
+export brownden
+
+"Brown and Dennis function"
+function brownden(n::Int=4, m::Int=20)
```
Comment (abelsiqueira) [inline src/brownden.jl]: `n` is not variable here.

**Comment by dpo** [comment]
Thanks. Fixed.

**Comment by abelsiqueira** [comment]
Thank you. I don't have write access, however.

**Comment by dpo** [comment]
Sorry about that. Fixed.

**Comment by abelsiqueira** [comment]
Done. :+1:

**Comment by dpo** [comment]
Thank you!

**Pull request merged**